tests/integration/security/util/framework.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package util
+
+import (
+	"istio.io/istio/pilot/pkg/model"
+	"istio.io/istio/pkg/test/framework/components/echo"
+	"istio.io/istio/pkg/test/framework/components/galley"
+	"istio.io/istio/pkg/test/framework/components/namespace"
+	"istio.io/istio/pkg/test/framework/components/pilot"
+)
+
+func EchoConfig(name string, ns namespace.Instance, headless bool, annos echo.Annotations, g galley.Instance, p pilot.Instance) echo.Config {
```

Comment:
I would suggest just put this in `grpc_test.go`, it could be mush shorter if you define it as a lamda function right in the `TestRBACV2GRPC()`, not too much benefit of putting it in a separate file.

Reply:
Many places in security tests are using the same code to get echo instances. Putting in a separate file since this is not a grpc or rbac specific. I will refactor other tests to use this function.

Similar effort but for one test only: https://github.com/istio/istio/commit/2a6bf3671b26e2780a9b6e9de05b61a4fdca4d33#diff-142e7e2ec4553ec47315710067d9f4fbR241